### PR TITLE
InCanvasSearch: Horizontal scroll bar

### PR DESCRIPTION
**Pull request by aosyatnik**
### Purpose

Link to YouTrack:
[MAGN-7407](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7407) Add horizontal scroll bar for incanvas search results

Result:
image

Scrollbar appears only if mouse is over element and title of element is long.

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] The level of testing this PR includes is appropriate

### Reviewers

@pboyer 
@Benglin 

### FYIs

@lillismith

### CODE REVIEW

File: src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml [hunk 1]
```diff
@@ -23,8 +23,8 @@
             <Style x:Key="SearchMemberStyle"
                    BasedOn="{StaticResource MemberGroupMemberStyle}"
                    TargetType="{x:Type ListBoxItem}">
-                <EventSetter Event="PreviewMouseLeftButtonDown"
-                             Handler="OnMouseLeftButtonDown" />
+                <EventSetter Event="PreviewMouseLeftButtonUp"
+                             Handler="OnMouseLeftButtonUp" />
```
Comment (Benglin) [inline src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml]: I remember seeing this change for the drag-and-drop PR two weeks ago, hopefully this change doesn't break that (not sure why is this changed here).

Comment (aosyatnik) [inline src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml]: It's because of dragging scroll bar.
Member button is "under" scrollviewer. So, if we click on member button or scroll bar, actually we click on scrollviewer. That's why we check on Up event, what was the original sender. If it was button - create node. If it was Thumb - do not create node.

**Comment by Benglin** [comment]
Thanks for this, @aosyatnik. It generally looks and works great to me, if I have to say something, it would be the horizontal scroll bar size, it's small and quite hard for me to grab on (it generally just create a node instead of allowing me to scroll). That can be addressed separately though, LGTM!
